### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by friederbluemle**
Just a few minor README.md updates. It was still referring to `init()` in a few places - which was replaced with `onCreate` in the latest version.

### CODE REVIEW

**Comment by avluis** [comment]
Plenty of the edits are good, but why this change:
```java
[...] to //...
```
Making it easier for copying and pasting?

File: README.md [hunk 1]
```diff
@@ -114,16 +113,20 @@ There's no need to create one class for fragment anymore. :)
 No problem, just use this method and AppIntro will generate a new slide for you.
 
 ```java
-addSlide(AppIntroFragment.newInstance(title, description, image, background_colour));
+addSlide(AppIntroFragment.newInstance(title, description, image, backgroundColor));
```
Comment (avluis) [inline README.md]: Grap this bit in onCreate as well.

File: README.md [hunk 2]
```diff
@@ -22,29 +22,28 @@ repositories {
...
 
 Create a new **Activity that extends AppIntro**:
 
 ```java
-public class MyIntro extends AppIntro {
-
+public class IntroActivity extends AppIntro {
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
```
Comment (avluis) [inline README.md]: Don't annotate this just yet -- need to make a quick change on the library first.

Comment (friederbluemle) [inline README.md]: It looks like `AppIntroBase` already has this annotation: https://github.com/PaoloRotolo/AppIntro/blob/master/library/src/main/java/com/github/paolorotolo/appintro/AppIntroBase.java#L75

File: README.md (hunk 1 above)
Comment (friederbluemle) [inline README.md]: Not sure what you mean by that..

**Comment by friederbluemle** [comment]
> @friederbluemle 
> Plenty of the edits are good, but why this change:
> 
> [...] to //...
> Making it easier for copying and pasting?

Hah, that's just a minor change because `[...]` is not "valid" Java code. Anyway, I can revert if you want.

File: README.md (hunk 2 above)
Comment (avluis) [inline README.md]: Go ahead and do the same for `AppIntro`.

**Comment by avluis** [comment]
> > @friederbluemle 
> > Plenty of the edits are good, but why this change:
> > 
> > [...] to //...
> > Making it easier for copying and pasting?
> 
> Hah, that's just a minor change because `[...]` is not "valid" Java code. Anyway, I can revert if you want.

Nah, that's ok -- I prefer it this way as well -- proper code and less mistakes for others to make.

File: README.md (hunk 2 above)
Comment (avluis) [inline README.md]: Ah, good then, go ahead with the change.

File: README.md (hunk 1 above)
Comment (friederbluemle) [inline README.md]: Oh, got it! Sure, good suggestion :+1: I wrapped it as well.

**Comment by avluis** [comment]
I see the changes -- thank you for taking the time to contribute~

**Pull request merged**